docs/general/fixtures.md

```diff
@@ -0,0 +1,255 @@
+
+### get-signals-v1-private-roles.json
+
+Old Name: roles.json
+
+Endpoint: signals/v1/private/roles/
+
+Key: results
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/roles/' "Authorization:Bearer $SIA_TOKEN" | jq '.results' > get-signals-v1-private-roles.json
+
+### get-signals-v1-private-categories.json
+
+Old Name: categories_private.json
+
+Endpoint: signals/v1/private/categories/
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/categories/' "Authorization:Bearer $SIA_TOKEN" | jq > get-signals-v1-private-categories.json
+
+### get-signals-v1-private-departments.json
+
+Old Name: departments.json
+
+Endpoint: signals/v1/private/departments/
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/departments/' "Authorization:Bearer $SIA_TOKEN" | jq > get-signals-v1-private-departments.json
+
+### get-signals-v1-private-users.json
+
+Old Name: users.json
+
+Endpoint: v1/private/users/
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/users/' "Authorization:Bearer $SIA_TOKEN" | jq > get-signals-v1-private-users.json
+
+### get-signals-v1-private-users-id.json
+
+Notes: This fixture is pretty fragile since it needs a specific set of permissions the issue is when
+more then the required permissions are added (different type permissions) tests break badly
+
+Old Name: user.json
+
+Endpoint: v1/private/users/143
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/users/143' "Authorization:Bearer $SIA_TOKEN" | jq > get-signals-v1-private-users-id.json
+
+### get-signals-v1-private-signals.json
+
+Old Name: incidents.json
+
+Endpoint: v1/private/signals
+
+Key: results
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/signals/' "Authorization:Bearer $SIA_TOKEN" | jq '.results' > get-signals-v1-private-signals.json
+
+### get-signals-v1-private-signals-id.json
+
+Old Name: incident.json
+
+Endpoint: v1/private/signals/123
+
+Key: results
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/signals/123' "Authorization:Bearer $SIA_TOKEN" | jq '.results' > get-signals-v1-private-signals-id.json
+
+### get-signals-v1-private-categories-id.json
+
+Old name: category.json
+
+Endpoint: v1/private/categories/23
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/categories/23' "Authorization:Bearer $SIA_TOKEN" | jq '.results' > get-signals-v1-private-categories-id.json
+
+### get-signals-v1-private-departments-id.json
+
+Old name: department.json
+
+Endpoint: v1/private/department/6
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/departments/6' "Authorization:Bearer $SIA_TOKEN" | jq > get-signals-v1-private-departments-id.json
+
+### get-signals-v1-private-categories-id-history.json
+
+Old name: history.json
+
+Endpoint: v1/private/categories/23/history.json
+
+Note: There is currently no history on acc
```

Comment:
I'm pretty sure there is, but it's not using a fixture.

Reply:
I should have phrased this differently, will come back to you with what I meant to say.